plasmapy/utils/pytest_helpers.py

```diff
@@ -874,3 +874,5 @@ def _prepare_input(param_name, param_default, insert_some_nans, insert_all_nans,
             astrohelper.assert_quantity_allclose(res_0d, result_1d[idx][3])
             astrohelper.assert_quantity_allclose(res_0d, result_2d[idx][1, 1])
             astrohelper.assert_quantity_allclose(res_0d, result_3d[idx][0, 1, 1])
+
+del Any, Callable, Dict
```

Comment:
Bloody hell I should have noticed this earlier >_< I know why this is erroring Let's revert this commit in this PR and I'll fix it collectively in #596.